packages/contracts-bedrock/src/L2/CrossL2Inbox.sol

```diff
@@ -0,0 +1,157 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.24;
```

Comment:
`0.8.25` just released, perhaps we could move to that version, it uses MCOPY over loops of MSTORE

Reply:
good idea!